test-clients/src/main/java/com/hedera/services/yahcli/commands/schedules/ScheduleCommand.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2021-2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.services.yahcli.commands.schedules;
+
+import com.hedera.services.yahcli.Yahcli;
+import java.util.concurrent.Callable;
+import picocli.CommandLine;
+import picocli.CommandLine.HelpCommand;
+import picocli.CommandLine.ParentCommand;
+
+@CommandLine.Command(
+        name = "schedule",
+        subcommands = {HelpCommand.class, SignCommand.class},
+        description = "Performs scheduler operations")
+public class ScheduleCommand implements Callable<Integer> {
+    @ParentCommand Yahcli yahcli;
+
+    @Override
+    public Integer call() throws Exception {
```

Comment:
Why are we throwing Exception instead of ParameterException?

Reply:
We probably just used IntelliJ to auto-implement the `Callable` interface, which [declares `Exception` here](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/Callable.html#call--).

Reply:
Since this can only ever throw ParameterException, IMO it would be better to have it throw the specific subclass

Reply:
ok will change it